ui/components/ContributeForm.tsx

```diff
@@ -213,18 +213,34 @@ const ContributeFormContents: FunctionComponent<ContributeFormProps> = ({
           inputMode="decimal"
           placeholder="Contribute..."
           accent={
-            expectedFundingTokensReceived
-              ? `You will receive about ${prettyPrintDecimal(
-                  expectedFundingTokensReceived
-                )} ${tokenSymbol}`
-              : undefined
+            <>
+              {!!watchContribution && watchContribution > maxContribution && (
+                <>
+                  The campaign is only {prettyPrintDecimal(maxContribution)}{" "}
+                  {payToken.symbol} short of reaching their funding goal!{" "}
+                  <Button
+                    bare
+                    className="inline underline"
+                    onClick={() =>
+                      doContribution({ contribution: maxContribution })
+                    }
+                  >
+                    Click here to get them to the finish line.
```

Comment:
Think it would be nice to keep the error text here otherwise might be a little unclear why they're seeing this text. How about "Campaigns can't be funded past their funding goal. Fund the remaining amount instead."

Reply:
Yes! This error is super unclear whoops. I think let's also just change the action button to "Fund remaining" instead of adding an inline text button and disabling the "Support this campaign" button. Might as well use the button already there since it shares a very similar purpose...

How about this:
<img width="759" alt="Screen Shot 2022-03-07 at 1 43 51 AM" src="https://user-images.githubusercontent.com/6721426/157006737-b4cb9568-b4df-46ef-8a72-6cb41005138e.png">

And when the wallet has insufficient balance:
<img width="754" alt="Screen Shot 2022-03-07 at 1 44 50 AM" src="https://user-images.githubusercontent.com/6721426/157006886-9bef8dd8-44fe-4a62-9c16-54bd4c4d966f.png">

Also, somewhat related and unrelated, added an insufficient funds error so they can see how much they need:
<img width="763" alt="Screen Shot 2022-03-07 at 1 45 43 AM" src="https://user-images.githubusercontent.com/6721426/157007031-fa091bfb-afb1-4f1e-9bd1-1ef7f1b763b2.png">